borsh/tests/test_schema_conflict.rs

```diff
@@ -0,0 +1,177 @@
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
I believe schema is not supported in no-std environment, so you may not pretend that it can and remove all the `std` guards and just disable this test in no-std.

Reply:
I was wrong here. Closing it.